test/dlc_tests/sanity/test_pt_s3.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+from invoke.context import Context
+from packaging.version import Version
+
+from test.test_utils import (
+    get_container_name,
+    get_framework_and_version_from_tag,
+    get_processor_from_image_uri,
+    run_cmd_on_container,
+    start_container,
+)
+
+
+@pytest.mark.model("N/A")
+def test_pt_s3_sanity(pytorch_training):
+    """
+    Check that the internally built PT S3 binary is properly installed.
+    :param pytorch_training: framework fixture for pytorch training
+    """
+    _, framework_version = get_framework_and_version_from_tag(pytorch_training)
+    if Version(framework_version) == Version("1.5.1") and get_processor_from_image_uri(pytorch_training) == "gpu":
```

Comment:
This should be `Version(framework_version) <= Version("1.5.1")`, because everything less than PT 1.6 is too old.